src/PreCICE.jl

```diff
@@ -327,8 +327,13 @@ Return true if the mesh is already used.
 
 """
 function hasData(dataName::String, meshName::String)::Bool
-    ans::Integer =
-        ccall((:precicec_hasData, "libprecice"), Cint, (Ptr{Int8}, Ptr{Int8}), dataName, meshName)
+    ans::Integer = ccall(
+        (:precicec_hasData, "libprecice"),
+        Cint,
+        (Ptr{Int8}, Ptr{Int8}),
+        dataName,
+        meshName,
+    )
```

Comment:
Is this a formatting change? Similar changes are seen below, and it seems that nothing is changing code-wise.